frontend/src/services/effects.ts

```diff
@@ -11,25 +11,34 @@ import { parseSerializedServiceArea } from '../utils/formatters'
 import { boundingBox, representativePointsToGeoJSON } from '../utils/geojson'
 import { equals } from '../utils/list'
 import { getPropCaseInsensitive } from '../utils/serializers'
-import { getAdequacies, getCensusData, getRepresentativePoints, isPostGeocodeSuccessResponse, postGeocode } from './api'
+import {
+    getAdequacies, getCensusData, getRepresentativePoints,
+    getStaticAdequacies, getStaticDemographics, getStaticRPs, isPostGeocodeSuccessResponse,
+    postGeocode
+} from './api'
 import { Store } from './store'
 
+/**
+ * Determine whether or not to use the static behaviours or the dynamic ones.
+ */
+const appIsStatic = CONFIG.staticAssets.appIsStatic
+
 export function withEffects(store: Store) {
   /**
    * Update representative points when serviceAreas change.
    */
   store
     .on('serviceAreas')
     .subscribe(async serviceAreas => {
+      const selectedDataset = store.get('selectedDataset')
 
-      if (serviceAreas.length === 0) {
-        store.set('representativePoints')([])
-      }
-
-      let points = await getRepresentativePoints({ service_area_ids: serviceAreas })
+      // Get representative points.
+      const points = appIsStatic ? await getStaticRPs(selectedDataset) :
+        await getRepresentativePoints({ service_area_ids: serviceAreas })
 
       // Get census information at the service area level.
-      let censusData = CONFIG.is_census_data_available ? await getCensusData({ service_area_ids: serviceAreas }) : {}
+      const censusData = appIsStatic ? await getStaticDemographics(selectedDataset) :
```

Comment:
Let's keep this independent no?

Reply:
I'm not sure I understand what you mean. Can you explain?

Reply:
You need to check `CONFIG.is_census_data_available` before `getStaticDemographics(selectedDataset)` as well

Reply:
Oh really? Why?

Reply:
The static site does have demographics data though

Reply:
For these datasets, but maybe not for others in the future. We should keep the behavior consistent.

Reply:
I follow. I'll submit that change now.

Reply:
Done.